nemo_text_processing/g2p/modules.py

```diff
@@ -579,12 +585,16 @@ def __call__(self, text):
         ' ', 'S', 't', 'o', 'r', 'e', ',', ' ', 'mai3', 'le5', 'yi2',
         'ge4', 'i', 'P', 'h', 'o', 'n', 'e', '。']
         """
-        pinyin_seq = self._lazy_pinyin(
-            text,
-            style=self._Style.TONE3,
-            neutral_tone_with_five=True,
-            errors=lambda en_words: [letter for letter in en_words],
-        )
+        pinyin_seq = []
+        # Cut sentences into words to improve polyphone disambiguation
+        words_list = list(jieba.cut(text))
```

Comment:
associated with relaxing hardcoded word segmenter. Please change this line according the changes above.